examples/audio_classifier/raspberry_pi/classify.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 The MediaPipe Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Main scripts to run audio classification."""
+
+import argparse
+import time
+
+from mediapipe.tasks import python
+from mediapipe.tasks.python.audio.core import audio_record
+from mediapipe.tasks.python.components import containers
+from mediapipe.tasks.python import audio
+from utils import Plotter
+
+
+def run(model: str, max_results: int, score_threshold: float,
+        overlapping_factor: float) -> None:
+  """Continuously run inference on audio data acquired from the device.
+
+  Args:
+    model: Name of the TFLite audio classification model.
+    max_results: Maximum number of classification results to display.
+    score_threshold: The score threshold of classification results.
+    overlapping_factor: Target overlapping between adjacent inferences.
+  """
+
+  if (overlapping_factor <= 0) or (overlapping_factor >= 1.0):
```

Comment:
This should be an overlapping_factor < 0; we should support 0 overlap to do no overlap stream at all. We can also clarify the error to say 0.0 and 0.9 (1 doesn't make sense because that's just the original stream done twice)